docs/general/builtins/scalar-functions.rst

```diff
@@ -2593,6 +2593,32 @@ An alias for :ref:`scalar-array_upper`.
     SELECT 1 row in set (... sec)
 
 
+.. NOTE::
+
+    For compatibility with PostgreSQL, :ref:`array_length <scalar-array_length>`
+    will return ``NULL`` on both empty arrays and ``NULL`` values.
+
+::
+
+    cr> select array_length([]::int[], 1) AS len;
```

Comment:
this one overlaps with https://github.com/crate/crate/pull/12702 - should keep only one of them?

Reply:
Thanks. I've removed the amendment to `scalar-functions.rst` completely.